specification/batch/data-plane/Microsoft.Batch/stable/2022-01-01.15.0/BatchService.json

```diff
@@ -17176,7 +17232,7 @@
         },
         "lowPriority": {
           "$ref": "#/definitions/NodeCounts",
-          "title": "The number of low-priority Compute Nodes in each state."
+          "title": "The number of low priority Compute Nodes in each state."
```

Comment:
Not sure why we made this change as our docs use "low-priority" with a hyphen. That said, see my other comment on using "Spot/low-priority" which is what we landed on for spot support.